products.css

```diff
@@ -0,0 +1,65 @@
+* {
+  font-family: "Inter", sans-serif;
+}
+
+.products-layout {
+  margin-top: 67px;
+  display: grid;
+  grid-template-columns: 250px 1fr;
+  height: 100%;
+  width: 100%;
+}
+
+.filters-container {
+  position: fixed;
+  width: 250px;
+  padding: 30px 20px 100px 25px;
+  height: 100%;
+  background-color: var(--primary-color);
+  overflow: auto;
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
```

Comment:
do the changes here

Reply:
i dont think it matters for small sizes but will change it